conformance-test/retryStrategy.ts

```diff
@@ -19,12 +19,14 @@ import * as path from 'path';
 import * as uuid from 'uuid';
 import pLimit = require('p-limit');
 import * as assert from 'assert';
+import fetch from 'node-fetch';
```

Comment:
does this introduce a dev dependency on node-fetch?

Reply:
We already had `node-fetch` listed as a dev dependency in the `package.json` which is why I went with it as opposed to the `https` lib in node.